app/graphql/mutations/create_street_art.rb

```diff
@@ -0,0 +1,22 @@
+class Mutations::CreateStreetArt < Mutations::BaseMutation
+  argument :latitude, String, required: false
+  argument :longitude, String, required: false
+  argument :address, String, required: false
+  argument :city, String, required: false
+  argument :state, String, required: false
+  argument :zipcode, String, required: false
+  argument :image_urls, String, required: true
+  argument :description, String, required: false
+  argument :artist_name, String, required: false
+  argument :art_name, String, required: false
+  argument :instagram_handle, String, required: false
+  argument :favorite, Boolean, required: false
+  argument :visited, Boolean, required: false
+  argument :user_id, Integer, required: true
```

Comment:
All are now set to `false` because the values will be filled in and validated at the next step which is in the StreetArt model